src/index.html

```diff
@@ -1,13 +1,311 @@
+            </p>
+          </section>
+
+          <section class="statistic" id="blog">
+            <div class="statistic__section">
+              <h2 class="statistic__title">2015</h2>
+              <p class="statistic__text">Foundation year</p>
+            </div>
+
+            <div class="statistic__section">
+              <h2 class="statistic__title">23</h2>
+              <p class="statistic__text">Stores</p>
+            </div>
+
+            <div class="statistic__background"></div>
+
+            <div class="statistic__section">
+              <h2 class="statistic__title">155</h2>
+              <p class="statistic__text">Companies</p>
+            </div>
+
+            <div class="statistic__section">
+              <h2 class="statistic__title">10K+</h2>
+              <p class="statistic__text">Clients</p>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer" id="footer">
+      <div class="footer__container">
+        <div class="footer__content">
+
+          <section class="footer__containerLogo">
+            <a href="#">
+              <img
+              src="images/logo-footer.png"
+              class="footer__logo"
+              alt="Bakery logo"
+              >
+            </a>
+          </section>
+
+          <section class="footer__menu">
+            <p class="footer__text-gray">Menu</p>
+            <a class="footer__link" href="#products">Products</a>
+            <a class="footer__link" href="#about">About us</a>
+            <a class="footer__link" href="#statistic">Blog</a>
+            <a class="footer__link" href="#footer">Terms & Privacy</a>
+          </section>
+
+          <section class="footer__contacts">
+            <h2 class="footer__title">Contact us</h2>
+          </section>
+
+          <section class="footer__numbers">
+            <p class="footer__text-gray">Call us</p>
+            <a class="footer__link" href="tel:+16295550129">(629) 555-0129</a>
+            <a class="footer__link" href="tel:+16035550123">(603) 555-0123</a>
+            <a class="footer__link" href="tel:+17045550127">(704) 555-0127</a>
+          </section>
+
+          <section class="footer__email">
+            <p class="footer__text-gray">Email us</p>
+            <a class="footer__link" href="mailto:bakerlab@example.com">bakerlab@example.com</a>
+          </section>
+
+          <section class="footer__social">
+            <a
+              href="https://www.facebook.com/"
+              class="footer__icon icon--facebook"
+              target="_blank">
+            </a>
+            <a
+              href="https://www.instagram.com/"
+              class="footer__icon icon--instagram"
+              target="_blank">
+            </a>
+            <a
+              href="https://twitter.com/"
+              class="footer__icon icon--twitter"
+              target="_blank">
```

Comment:
Need to fix it everywhere
```suggestion
            <a
              href="https://twitter.com/"
              class="footer__icon icon--twitter"
              target="_blank"
            >
```